packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -79,6 +80,13 @@ export interface IPortfolio extends cdk.IResource {
    */
   addProduct(product: IProduct): void;
 
+  /**
+   * Add notifications for supplied topics on the provisioned product.
+   * @param product A service catalog product.
+   * @param topics A list of SNS Topics to receive notifications on events related to the provisioned product.
+   */
+  addEventNotifications(product: IProduct, topics: sns.ITopic[], options?: CommonConstraintOptions): void;
```

Comment:
1. sure - I liked the notify verbiage but didn't like what I had for what would be the stackEvents part.  I also thought maybe something like `subscribeToEvents`.
2. I got my guidance before based on the original construct which took a list.  I don't have many good examples, but using the singular version might be easier here if that hypothesis is true.  For one it simplifies the code since we don't need the for loop.  The only concern I had was that if you had more than one topic and more than one portfolio-product association it would get very not DRY.  Adding just the singular version first sounds fine.

Reply:
changed this to singular.